lab1/app.css

```diff
@@ -0,0 +1,79 @@
+.vertical-menu {
+    overflow: scroll;
+    background-color: #000000;
+    padding: 20px 10px;
+    width: 1000px;
```

Comment:
highly recommend to not provide `px` based unit for width, it may limit the ability for you to make website responsive.